generators/pattern-library/configure.js

```diff
@@ -111,19 +110,69 @@ function init( options ) {
 
     // **** SETTINGS ****
 
-    if ( settings.dest ) {
+    if ( settings.dest && settings.dest === 'none' ) {
+
+        settings.dest = null;
+
+    }
+    else if ( settings.dest ) {
+
         settings.dest = path.resolve( settings.cwd, settings.dest );
+
+    }
+    else {
+
+        error = error || [];
+        error.push({
+            'key': 'settings.dest',
```

Comment:
`key` is a valid identifier, no need for quotes. same with within the rest of this file